### PR TITLE
as: add page

### PR DESCRIPTION
**Pull request by agnivade**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

**Comment by agnivade** [comment]
New page after a long time !

File: pages/linux/as.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+- Assemble a file, writing the output to a.out:
+
+`as {{file.s}}`
+
+- Assemble the output to a given file:
+
+`as {{file.s}} -o {{out.o}}`
+
+- Skip whitespace and comment preprocessing, leading to faster output. (Should only be used for trusted compilers):
```
Comment (sbrl) [inline pages/linux/as.md]: `faster output`? Does this mean it runs faster (i.e. faster operation), or that the output binary is more optimised?

Comment (agnivade) [reply pages/linux/as.md]: faster operation. Does it sound confusing ? I thought it was clear enough :thinking:

Comment (sbrl) [reply pages/linux/as.md]: Yeah, it does a bit

Comment (pxgamer) [reply pages/linux/as.md]: I think it's pretty clear actually. :thinking:

Comment (pxgamer) [reply pages/linux/as.md]: @sbrl, any thoughts on what this could be changed to in order to make it clearer? 👍

Comment (sbrl) [reply pages/linux/as.md]: Perhaps `output that operates faster` would work better, @pxgamer?

Comment (agnivade) [reply pages/linux/as.md]: Maybe I should just remove that bit altogether. It is just extra information which will anyways becomes apparent once the user runs it.

Comment (sbrl) [reply pages/linux/as.md]: Erm that would imply that the output is generated faster, which comflicts with your earlier statement of

>  operation. Does it sound confusing ? I thought it was clear enough thinking

Comment (agnivade) [reply pages/linux/as.md]: You said - `Does this mean it runs faster (i.e. faster operation)`. I said yes. This is implying the same. Output is generated faster means it runs faster.

Comment (sbrl) [reply pages/linux/as.md]: I'm confused. Nevermind then :P

Comment (agnivade) [reply pages/linux/as.md]: No worries. I will update to the new description :)

**Pull request merged**